internal/ethapi/api.go

```diff
@@ -80,6 +80,19 @@ func (s *PublicEthereumAPI) MaxPriorityFeePerGas(ctx context.Context) (*hexutil.
 	return (*hexutil.Big)(tipcap), err
 }
 
+type feeHistoryResults struct {
+	FirstBlock   rpc.BlockNumber
+	Reward       [][]*big.Int //hexutil.Big
+	BaseFee      []*big.Int   //hexutil.Big
```

Comment:
Why not use the `hexutil.Big`?

Reply:
I wanted to but then forgot about it. Did it now.